CoreFindView/src/au/gov/asd/tac/constellation/views/find/FindViewHelpProvider.java

```diff
@@ -45,7 +45,7 @@ public Map<String, String> getHelpMap() {
         final String findModulePath = ".." + sep + "ext" + sep + "docs" + sep + "CoreFindView" + sep + "src" + sep + "au" + sep + "gov" + sep + "asd"
                 + sep + "tac" + sep + CODEBASE_NAME + sep + "views" + sep + "find" + sep;
 
-        map.put("au.gov.asd.tac.constellation.views.find", findModulePath + "find-view.md");
+        map.put("au.gov.asd.tac.constellation.views.find.FindViewTopComponent", findModulePath + "find-view.md");
```

Comment:
The corresponding toc.xml file needs to be updated to match otherwise when the help button is clicked it won't find the page

The reference to the link in the button code will also need to be updated to match